_posts/2018-11-30-Rates-Rebate.md

```diff
@@ -0,0 +1,453 @@
+
+__System__
+- The process is very __manual__ and involves __costly double handling__. Applicants must fill in the same information and apply every year even when their information does not change.
+- Preventable __cost of implementation to councils__ and wait times for payments back to councils from DIA
+- __Prescriptive legislation__ preventing an end-to-end digital experience
+
+## The discovery<a name = "Go-to-Discovery"></a>
+
+In​ ​September​ 2017 ​the ​team​ ​kicked​ ​off​ ​a​ ​three-week​ ​discovery​ ​sprint,​ ​exploring​ ​how​ ​subsidies​ ​are provided​ ​to​ ​New​ ​Zealanders​ ​and​ ​what​ ​ ​that​ ​could look​ ​like​ ​in​ ​the​ ​future,​ ​with​ ​a​ ​focus​ ​on​ ​Rates Rebates.​ ​Rates​ ​Rebates​ ​are​ ​an​ ​entitlement​ ​that​ ​has​ ​been​ ​identified​ ​as​ ​a​ ​key​ ​pain​ ​point​ ​for​ ​applicants (low-income​ ​property​ ​owners​ ​for​ ​primary​ ​residences),​ ​as​ ​well​ ​as​ ​for​ ​Local​ ​and​ ​Central​ ​Government.
+
+The discovery sprint included service designers, user researchers and subject matter experts from the Department of Internal Affairs (the Service Innovation team as well as the Rates Rebates team), Auckland Council, Tauranga Council, Wellington Council, the Ministry for Social Development and the Inland Revenue Department with support from Assurity.
+
+## Week One - Kick off
+### Focus points
+The team identified the following focus points for the three week sprint.
+- __Understand​ ​and​ ​Complement​__ - Previous Rates Rebates insights focused on the process from an applicant perspective, our team will collect and analyse insight from multiple perspectives including Central and Local Government agencies. We are also looking for additional applicant types that have not been spoken to before and understand the size and demographics of the eligibility groups.
+
+- __Local​ ​and​ ​Global​ ​Picture__​ - We are exploring the local and global community and identifying similar concepts, prototypes and entitlement models that we can factor into our concept development.
+
+ - __Data​ ​Path​__ - Exploring the data path of the process, data ownership, authority and declaration and what this could mean for implementation of any automated concepts.
+
+- __Thinking​ ​Big/Parallel​ ​Value​__ - How could this be applied across all of government services.
+
+## Week Two - Customer research and journey mapping
+### Service blueprint
+Service Map The Rates Rebates Process touches many agencies. We mapped the journey to see touch points for rebates customers, front stage (agency work visible to the rebate customer), and backstage (agency work not visible to the rebate customer). It showed a very complex picture with a lot of pain across users, Councils and Central government agencies. Considering this entitlement is a maximum of $610 per year, it requires a lot of cost and time from all participants in the process to administer.
+![Service Blueprint](/assets/media/Rates-Rebates/RR2.png)
```

Comment:
Do you think the link below explains the image enough to people who cannot see images? Can it be cleare?